core/slice.go

```diff
@@ -374,6 +374,14 @@ func (sl *Slice) GetPendingHeader() (*types.Header, error) {
 	return sl.phCache[sl.pendingHeader].Header, nil
 }
 
+func (sl *Slice) GetSubManifest(blockHash common.Hash) (types.BlockManifest, error) {
+	header := sl.hc.GetHeaderByHash(blockHash)
+	if header == nil {
+		return nil, fmt.Errorf("block not found")
```

Comment:
typically we use errors.New("block not found") don't know if this will break error catching

Reply:
fixed: [6ab0fd7](https://github.com/dominant-strategies/go-quai/pull/463/commits/6ab0fd74272c6ea3aa2edccd1134620b4cd3143a)